framework/src/components/README.md

```diff
@@ -0,0 +1,28 @@
+# Components
+
+## Description
+
+Components are shared objects within the [controller](../controller/README.md) layer which any [module](../modules/README.md) can utilize.
+Components can use [channels](../modules/README.md#module-communication) if required for implementation behavior.
+The following components are available currently.
+
+### Cache
+
+This component provides basic caching capabilities, generic enough for any module to use if required.
+
+### Logger
+
+Logger is responsible for all application-level logging activity and logs everything in JSON format.
```

Comment:
does it log in JSON format now?

Reply:
We didn't include it in 1.6.0. We should do it in 2.0.

 That's something we would have to amend in the LIP if 1.6.0 is about to close the objective.

Reply:
I deleted the false part for now.